addons/website/static/src/scss/website.editor.ui.scss

```diff
@@ -1,813 +1,803 @@
+        z-index: $zindex-modal - 9;
+        position: fixed;
+        top: 0px;
+        right: 0px;
+        width: auto;
+        white-space: nowrap;
+
+        background-color: rgba(0, 0, 0, 0);
+        transition: background-color 400ms $o-we-md-ease 0s;
+        font-family: $o-we-font-family;
 
-        .btn {
-            height: 100%;
+        form.navbar-form {
+            height: $o-navbar-height;
+            z-index: 1060;
             margin: 0;
-            padding: 10px;
-            line-height: 1.2;
-            font-size: 13px;
-            font-family: $o-we-font-family;
+            padding: 0;
+            @include o-position-absolute($top: -1px, $right: -$o-we-sidebar-width);
+            right: 0;
+            transition: right 0.4s $o-we-md-ease 0s;
+            border-left: 1px solid $o-we-color-divider;
+            background-color: inherit;
 
-            transition: all 0.3s ease 0s;
-            border: none;
-            border-radius: 0;
+            .btn-group {
+                height: 100%;
+            }
 
-            .fa {
-                margin-right: $grid-gutter-width/4;
+            .btn {
+                height: 100%;
+                margin: 0;
+                padding: 10px;
+                line-height: 1.2;
+                font-size: 13px;
+                font-family: $o-we-font-family;
 
-                &.fa-times {
-                    color: $o-we-color-danger;
+                transition: all 0.3s ease 0s;
+                border: none;
+                border-radius: 0;
+
+                .fa {
+                    margin-right: $grid-gutter-width/4;
+
+                    &.fa-times {
+                        color: $o-we-color-danger;
+                    }
                 }
-            }
 
-            &.btn-primary {
-                @include button-variant($o-brand-odoo, $o-brand-odoo);
-            }
+                &.btn-primary {
+                    @include button-variant($o-brand-odoo, $o-brand-odoo);
+                }
 
-            &.btn-secondary {
-                @include button-variant($o-we-color-dark, $o-we-color-dark);
+                &.btn-secondary {
+                    @include button-variant($o-we-color-dark, $o-we-color-dark);
 
-                &:hover {
-                    background: rgba(black, 0.5);
+                    &:hover {
+                        background: rgba(black, 0.5);
+                    }
+                }
+
+                &:focus,
+                &:active,
+                &:focus:active {
+                    outline: none;
                 }
             }
 
-            &:focus,
-            &:active,
-            &:focus:active {
-                outline: none;
+            .dropdown-menu {
+                left: auto;
+                right: 0;
             }
         }
+    }
 
-        .dropdown-menu {
-            left: auto;
-            right: 0;
+    #wrapwrap {
+        height: calc(100vh - #{$o-navbar-height});
```

Comment:
Mais enfin... :cry: That was not there in 12.0 right ? Why changing what was done and which worked by a calc :expressionless:  ?

Reply:
(just a thought outside the scope of this PR)

Reply:
Yes not this pr. And it is introduced because now, we edit a copy of the Dom... Some display are different :-s